CRM/Admin/Form/MailSettings.php

```diff
@@ -77,7 +77,7 @@ public function buildQuickForm() {
 
     $this->add('password', 'password', ts('Password'), ['autocomplete' => 'off']);
 
-    $this->add('text', 'source', ts('Source'), $attributes['source']);
+    $this->add('text', 'source', ts('Mail Folder'), $attributes['source']);
```

Comment:
Seems like a good change, but Mail Folder feels redundant here, where Folder would definitely be sufficient in the context.

Reply:
Thanks for the feedback. I'm happy with leaving it as Mail Folder because it is more specific.